pageserver/src/waldecoder.rs

```diff
@@ -263,6 +267,7 @@ pub struct DecodedBkpBlock {
     pub rnode_spcnode: u32,
     pub rnode_dbnode: u32,
     pub rnode_relnode: u32,
+    // Note that we have a few special forknum values for non-rel files.
```

Comment:
A rust enum for the non-rel files would probably be readable.

(I'm still a bit sceptical of having special handling for SLRU files in the page server, as it requires more Postgres changes, but that's another topic)